runners/direct-java/build.gradle

```diff
@@ -70,7 +69,6 @@ dependencies {
   shadow library.java.args4j
   provided library.java.hamcrest_core
   provided library.java.junit
-  testRuntime project(path: ":sdks:java:harness")
```

Comment:
+1, this was a long time pain point since this was part of the classpath and not an arg to TestPipeline that allowed the portable direct runner to launch the SDK harness as a separate process.